src/mixer.h

```diff
@@ -109,6 +111,29 @@ class Mixer : public cyclus::Facility {
 
   //// A policy for sending material
   cyclus::toolkit::MatlSellPolicy sell_policy;
+
+  private:
+  /////////// position toolkit ///////////
+  #pragma cyclus var { \
+    "default": 0.0, \
+    "uilabel": "Geographical latitude in degrees as a double", \
+    "doc": "Latitude of the agent's geographical position. The value should " \
+           "be expressed in degrees as a double." \
+  }
+  double latitude;
+
+  #pragma cyclus var { \
+    "default": 0.0, \
+    "uilabel": "Geographical longitude in degrees as a double", \
+    "doc": "Longitude of the agent's geographical position. The value should " \
+           "be expressed in degrees as a double." \
+  }
+  double longitude;
+
+  cyclus::toolkit::Position coordinates;
+
+  /// Records a reactors latitude and longitude to the output db
```

Comment:
"a reactors" should be "an agent's"
change this everywhere. (12 times)

Reply:
Thank you for the correction. I'll make the changes right now!